forte/data/data_pack.py

```diff
@@ -172,7 +170,9 @@ def __init__(self, pack_name: Optional[str] = None):
         self._data_store: DataStore = DataStore()
         self._entry_converter: EntryConverter = EntryConverter()
         self.image_annotations: List[ImageAnnotation] = []
-        self.grids: List[Grids] = []
+        self.payloads: List[np.ndarray] = []
```

Comment:
Here is an unused variable. And more, this make a new assumption that we store a list of payload in the format as `np.ndarray`. Let's not do this now (at least not in this PR).